web/api/routes.go

```diff
@@ -16,26 +16,26 @@ func (ar *Router) initRoutes() {
 	apiMiddlewares := ar.middleware.With(ar.DumpRequest(), ar.AppID())
 
 	// setup root routes
-	ar.router.HandleFunc("/{ping:ping\\/?}", ar.HandlePing()).Methods("GET")
+	ar.router.HandleFunc(`/{ping:ping/?}`, ar.HandlePing()).Methods("GET")
```

Comment:
I wonder, what's difference does this notation make?

Reply:
To my mind, this looks more concise and clear due to the absence of slash escaping.